src/components/BlockCreateModal.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <p-modal v-model:showModal="internalShowModal" class="block-create-modal" :class="modalClass" title="Add a new block">
+  <p-modal v-model:showModal="internalShowModal" class="block-create-modal" :class="classes" title="Add a new block">
```

Comment:
```suggestion
  <p-modal v-model:showModal="internalShowModal" class="block-create-modal" :class="classes" :title="title">
```

Reply:
Whoops, I actually means to keep the static title and use the heading in the form itself. I'll delete the unused title computed. 👍